### ISSUE TITLE
Update aiohttp to version >= 3.8.5

### ISSUE THREAD

**Issue by nico-stefani** [wazuh/wazuh#18213]
|Wazuh version|Component|
|---|---|
| 4.7.0 | API/Framework |

## Description

Due to a vulnerability detected in `aiohttp` library prior to `3.8.1`, it needs to be updated at least to version `3.8.5` or later.

Scan result:

```
{
    "package_name": "aiohttp",
    "package_version": "3.8.1",
    "package_affected_version": "<=3.8.4",
    "vuln_description": "Aiohttp 3.8.5 includes a fix for CVE-2023-37276: Sending a crafted HTTP request will cause the server to misinterpret one of the HTTP header values leading to HTTP request smuggling.\r\nhttps://github.com/aio-libs/aiohttp/commit/9337fb3f2ab2b5f38d7e98a194bde6f7e3d16c40\r\nhttps://github.com/aio-libs/aiohttp/security/advisories/GHSA-45c4-8wx5-qw6w",
    "safety_id": "59725"
}
```
- [AIOHTTP versions changelog](https://docs.aiohttp.org/en/stable/changes.html)

## Checks

The following elements have been updated or reviewed (should also be checked if no modification is required):
- [ ] Tests (unit tests, API integration tests).
- [ ] Changelog.
- [ ] Documentation.
- [ ] Integration test mapping (using `api/test/integration/mapping/_test_mapping.py`).

**Comment by EduLeon12** [comment]
## Issue Update

During [Week 44: Framework vulnerability scan](https://github.com/wazuh/internal-devel-requests/issues/429) the targeted version should changed to version higher or equal to 3.8.6.

**Comment by fdalmaup** [comment]
# Issue Update

The `aiohttp` dependency was updated for Wazuh `4.8.0` in #13265:

https://github.com/wazuh/wazuh/blob/24e541b489e11578e7979bb04ecbf6bc3b5f3497/framework/requirements.txt#L1

Since there are no available `3.8.5` or higher version wheels for CentOS 6 and the Wazuh `4.7.1` code is soon to be delivered, there is no time to analyze and execute the dependency upgrade for said version.
